backend/app/routers/experiment_templates.py

```diff
@@ -134,23 +245,26 @@ def find_specific_experiment_templates(
         if pagination is not None
         else {}
     )
+    if len(search_query) > 0:
+        search_conditions.append(Text(search_query))
+
+    if user is None and only_mine:
+        # Authentication required to see your experiment templates
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            detail="This endpoint requires authorization. You need to be logged in.",
+            headers={"WWW-Authenticate": "Bearer"},
+        )
+    elif only_mine:
+        search_conditions.append(ExperimentTemplate.created_by == user["email"])
 
-    if user is None:
-        if only_mine or include_pending:
-            # Authentication required to see your experiment templates or pending experiment templates
-            raise HTTPException(
-                status_code=status.HTTP_401_UNAUTHORIZED,
-                detail="This endpoint requires authorization. You need to be logged in.",
-                headers={"WWW-Authenticate": "Bearer"},
-            )
-    else:
-        if only_mine:
-            search_conditions.append(ExperimentTemplate.created_by == user["email"])
-
-    if not include_pending:
-        search_conditions.append(ExperimentTemplate.approved == True)  # noqa: E712
     if only_finalized:
         search_conditions.append(ExperimentTemplate.state == TemplateState.FINISHED)
+        search_conditions.append(ExperimentTemplate.approved == True)  # noqa: E712
```

Comment:
Why merging finalized with approved? Additionally, only approved template can be finalized, so this would not be needed anyway, right?

Reply:
I decided not to distinguish between approved templates and finalized templates as there's currently no need to list the approved templates that haven't been finalized yet. In other words, you're only interested in displaying the templates that have been finalized and work properly.

And in terms of that second condition you have pointed out, I might've put it there just to be sure, but it indeed is not necessary and will be removed

Reply:
As admin I would like to list all non-approved templates. When requesting only non-finalized templates I might get approved templated that crashed during the image building.